pyramid_apispec/helpers.py

```diff
@@ -134,7 +134,8 @@ def add_pyramid_paths(
 
 
 def is_view(introspectable):
-    return introspectable.category_name == "views"
+    is_special_cornice = introspectable.title == 'function cornice.pyramidhook._fallback_view'
```

Comment:
Can we have that configurable - via keyword parameter like `custom_introspectables` = None, and then in `add_pyramid_paths` if it is None create a list with `['function cornice.pyramidhook._fallback_view']`.
This would allow for better flexibility.